addons/portal/static/src/js/portal_chatter.js

```diff
@@ -151,6 +151,7 @@ var PortalChatter = publicWidget.Widget.extend({
     _reloadChatterContent: function (data) {
         this.messageFetch();
         this._reloadComposer();
+        location.reload();
```

Comment:
Seems like too big of a change if the only issue is that the comment count are not updated. I would prefer a proper js fix for the master branch than reloading the whole page each time just for this in stable.

Otherwise we may as well not do anything dynamically.